corehq/apps/case_importer/templates/case_importer/import_cases.html

```diff
@@ -48,4 +95,71 @@
             </div>
         </div>
     </form>
+
+
+    <h2>Recent Uploads</h2>
+    <div id="recent-uploads" class="ko-template">
+        <!--ko if: state() == states.SUCCESS && !_.isEmpty(case_uploads())-->
+        <table class="table table-striped">
+            <tr>
+                <th class="col-md-2">{% trans "Status" %}</th>
+                <th class="col-md-1">{% trans "Case Type" %}</th>
+                <th class="col-md-1">{% trans "Uploaded by" %}</th>
+                <th class="col-md-6">{% trans "Details" %}</th>
+                <th class="col-md-2">{% trans "Time" %}</th>
+            </tr>
+            <!--ko foreach: case_uploads-->
+            <tr>
+                <td class="col-md-2">
+                    <!--ko if: task_status.state == $root.states.SUCCESS-->
+                        <!--ko if: _.isEmpty(task_status.result.errors)-->
+                        <span class="label label-success">{% trans "Success" %}</span>
+                        <!--/ko-->
+                        <!--ko if: !_.isEmpty(task_status.result.errors)-->
+                        <span class="label label-warning">{% trans "Success with warnings" %}</span>
+                        <!--/ko-->
+                    <!--/ko-->
+                    <!--ko if: task_status.state == $root.states.FAILED-->
+                    <span class="label label-danger">{% trans "Failed" %}</span>
+                    <!--/ko-->
+                    <!--ko if: task_status.state == $root.states.NOT_STARTED-->
+                    {% trans "Waiting to start" %}
+                    <!--/ko-->
+                    <!--ko if: task_status.state == $root.states.STARTED-->
+                    <div class="progress">
+                        <div class="progress-bar progress-bar-info"
+                             role="progressbar" aria-valuenow="60" aria-valuemin="0" aria-valuemax="100"
+                             data-bind="attr: {style: 'width: ' + task_status.progress.percent + '%;'}">
+                            <span><!--ko text: task_status.progress.percent --><!--/ko-->%
+                                {% trans "Complete" %}</span>
+                          </div>
+                    </div>
+                    <!--/ko-->
+                </td>
+                <td class="col-md-1">
+                    <!--ko text: case_type--><!--/ko-->
```

Comment:
why use a comment binding there instead of a text binding on the `td` or on a `span`?